Volume/Extensions/UIImage+Extension.swift

```diff
@@ -50,4 +50,17 @@ extension UIImage {
         )
     }
     
+    /// Returns the average color of this `UIImage`'s starting from the bottom with a height of 60. Gray if null.
+    var bottomAverageColor: UIColor? {
```

Comment:
nit: could be a more general (and maybe more useful too) function that takes in the desired `distanceFromBottom` and then computes the property described above. You can then just set `60` as some constant in your structs when you try to access this property/function

Reply:
Genius. Thanks!